.github/pull_request_template.md

```diff
@@ -16,3 +16,4 @@ For all contributors:
 - [ ] You have added tests (when appropriate).
 - [ ] You have added an entry in the CHANGELOG (when appropriate).
 - [ ] You have updated the README or other documentation to account for these changes (when appropriate).
+- [ ] You have either (a) made changes to sprocket.bio [if you are a Sprocket team member] or (b) have suggested any changes you _think_ need to be made to sprocket.bio in the description of your PR [if you are an external contributor].
```

Comment:
I think this belongs in `RELEASE.md` rather than the PR checklist, unless we have a place to stage the `sprocket.bio` changes pending a release.

Reply:
I've now suggested we start a convention where we keep a `next` branch in the Sprocket.bio repository that holds all changes. I'll now include in the `RELEASE.md` here to merge the `next` branch there during the release.